client/src/elm/MassiveDecks/Strings/Languages/Es.elm

```diff
@@ -606,19 +616,19 @@ translate _ mdString =
             [ ref (nounUnknownQuantity Spectator) ]
 
         SpectatorsDescription ->
-            [ Text "Jugadores en la partida espectando." ]
+            [ Text "Jugadores espectando la partida." ]
 
         Left ->
-            [ Text "Se ha ido" ]
+            [ Text "Se fueron" ]
```

Comment:
True, but I have translated it this way because it makes more sense to think of the translation as "Lista de usuarios que [Se fueron]", and it could be just one.

The other way, if you speak in singular and there is a long list of users who have left, it makes less sense.